lineage/figures/cartoons/gemcitabine.svg

```diff
@@ -1,93 +0,0 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?>
-<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN"
- "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
```

Comment:
Did you intend to delete this?

Reply:
Yes, actually, these are generated every time we run the whole figures. I know I had written something in the makefile to delete them each time, but not sure why it is not doing that.